src/main/resources/data/minebound/recipes/illuminant_steel_ingot_from_alloy_furnace.json

```diff
@@ -0,0 +1,19 @@
+{
+  "type": "minebound:alloy_furnace",
+  "ingredients": [
+    {
+      "item": "minecraft:glowstone_dust"
+    },
+    {
+      "item": "minebound:energized_iron_ingot"
+    },
+    {
+      "item": "minecraft:coal"
```

Comment:
use the coals tag:
`"tag":"#minecraft:coals"`

Reply:
If I remember correctly, you don't need the `#` for tags.